### ISSUE TITLE
Users can not select and delete a link when Segmentable == true

### ISSUE THREAD

**Issue by JohnMidity** [Blazor-Diagrams/Blazor.Diagrams#419]
When a normal (non-segmented) link is clicked by the user, it will be selected and the link can be deleted. But if `Segmentable==true` , a new vertex is added when the user clicks the line segment. The link is not selected, so it can also not be deleted.

Possible solution could be to select the link whenever the user adds/deletes/changes any vertex on the link. In normal use, the link is clicked, a vertex is added and then the users hits 'Delete' button to remove the link including the vertex .

**Comment by zHaytam** [comment]
What do you think about adding vertices by double-clicking instead of clicking? Then clicking would let you select the link

**Comment by JohnMidity** [comment]
Yes, I think that would be fine and can easily be explained to users.
